<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="NTlabel">
         <property name="text">
          <string>Neighborhood type:       </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="NTcomboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Interest points selection:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="IPScomboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="DFlabel">
         <property name="text">
          <string>Diagonal fraction</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="RNlabel">
         <property name="text">
          <string>Number of rings</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="RNlineEdit">
         <property name="placeholderText">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="HPlabel">
         <property name="text">
          <string>Harris parameter</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="DFlineEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="HPlineEdit">
         <property name="placeholderText">
          <string>0.02</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="RMDlabel">
         <property name="text">
          <string>Ring maxima detection</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="SPlabel">
         <property name="text">
          <string>Selection paramter</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="RMDlineEdit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="SPlineEdit">
         <property name="placeholderText">
          <string>0.02</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="OpenpushButton">
         <property name="text">
          <string>Open file</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="DetectpushButton">
         <property name="text">
          <string>Detect interest points</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="MyGLWidget" name="myGLWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>xRot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="rotXSlider">
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>16</number>
         </property>
         <property name="pageStep">
          <number>15</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>yRot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="rotYSlider">
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>16</number>
         </property>
         <property name="pageStep">
          <number>15</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>zRot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="rotZSlider">
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>16</number>
         </property>
         <property name="pageStep">
          <number>15</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header>myglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setZRotation(int)</slot>
    <slot>setYRotation(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rotXSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setXRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotYSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setYRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotZSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setZRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
